HLTrigger/HLTcore/interface/HLTPrescaleProvider.h

```diff
@@ -103,19 +103,32 @@ class HLTPrescaleProvider {
 private:
   void checkL1GtUtils() const;
   void checkL1TGlobalUtil() const;
+
   template <typename T>
   T convertL1PS(double val) const {
+    static_assert(std::is_same_v<T, double> or std::is_same_v<T, FractionalPrescale>,
+                  "\n\n\tPlease use convertL1PS<double> or convertL1PS<FractionalPrescale>"
+                  " (other types for L1T prescales are not supported anymore by HLTPrescaleProvider)"
+                  "\n\tconvertL1PS is used inside prescaleValues and prescaleValuesInDetail,"
+                  " so it might be necessary to specify template arguments for those calls,"
+                  "\n\te.g. prescaleValues<double, unsigned int>"
```

Comment:
I suppose this message needs to be updated as ```unsigned int``` is now no longer allowed for HLT?

Reply:
Yes, true, will do.